source/_templates/registrations/common/restart_manager.rst

```diff
@@ -0,0 +1,17 @@
+.. Copyright (C) 2020 Wazuh, Inc.
```

Comment:
Restart a wazuh-manager is a common task but it must not depend on the registration guide. The path of this file must be `source/_templates/common/restart_manager.rst`

Reply:
Done